husky:

  max_lin_vel: 1
  max_ang_vel: 2.5

fetch:

  max_lin_vel: 0.5
  max_ang_vel: 1.5
